src/main/java/rpgcharacters/userflow/ArchetypeMenu.java

```diff
@@ -0,0 +1,267 @@
+package rpgcharacters.userflow;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class ArchetypeMenu implements Menu {
+
+    private Connection conn;
+
+    private Scanner sc;
+
+    private List<String> options;
+
+    private final String ARCH_LIST = "List Archetypes";
+    private final String ARCH_NEW  = "Create a new Archetype";
+    private final String EXIT      = "Back to Admin Menu";
+
+    public ArchetypeMenu( Scanner sc ) {
+        this.sc = sc;
+        this.options = Arrays.asList( ARCH_LIST, ARCH_NEW, EXIT );
+    }
+
+    private void listArchetype() {
+        try {
+            // get every archetype, and the name of the quest that rewards it
+            String query = "SELECT *"
+                         + "FROM archetype";
```

Comment:
Need a space between `*` and `FROM`. Don't really need to have it on two separate lines